src/SFA.DAS.RoATPService.Application/Interfaces/IFatDataExportRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RoATPService.Application.Interfaces
+{
+    public interface IFatDataExportRepository
+    {
+        Task<IEnumerable<object>> GetFatDataExport();
```

Comment:
Could this return a strongly typed object?

Reply:
it could, but modelled it from the complete register one which uses object